packages/sonarwhal/src/lib/cli/analyze.ts

```diff
@@ -168,8 +169,12 @@ export default async (actions: CLIOptions): Promise<boolean> => {
         }
     }
 
-    const config = SonarwhalConfig.fromConfig(userConfig, actions);
-    const resources = resourceLoader.loadResources(config);
+    const config: SonarwhalConfig = SonarwhalConfig.fromConfig(userConfig, actions);
+    const resources: SonarwhalResources = resourceLoader.loadResources(config);
+
+    // Question: Should we track this after all packages has been installed?
```

Comment:
Want to track the results? There will be lots of sensible information like URLs, etc. that we will have to clean up and I don't think it's worth it.
If it is tracking that a scan has happened then the answer is yes (and we should track the configuration used).